src/lib.rs

```diff
@@ -105,10 +106,12 @@ unsafe extern "C" fn p_init(
 ) -> psa_status_t {
     let mut client = (*PARSEC_BASIC_CLIENT).write().expect("lock poisoned");
 
+    env_logger::init();
```

Comment:
Could you create a `log` feature on the crate that enables the `env_logger` dependency and initializes logging? That way we can simply compile with or without logs

Reply:
🎈 Done 🎈

Reply:
Thanks!